packages/store/src/Bytes.sol

```diff
@@ -6,41 +6,12 @@ pragma solidity >=0.8.21;
  * @notice Utility functions for bytes.
  */
 library Bytes {
```

Comment:
this library feels a little incomplete with half the bytes variants removed but 🤷‍♂️

maybe in the future, we move this into the `common` package or similar so that it represents a pure lib and not dependent on anything store related